### ISSUE TITLE
Group screenshots per device/destination

### ISSUE THREAD

**Issue by ntsh** [ChargePoint/xcparse#21]
Hi there!

Thank you for the project. I found this project useful for exporting screenshots with different device sizes for app store submission.

Currently, the screenshots are exported all in the same directory with a flat structure. 
It would be nice to group them into different folders with the destination name.

Eg: 

```
xcodebuild -workspace <workspace> -scheme <scheme> \
 -destination 'platform=iOS Simulator,name=iPhone 8 Plus' \
 -destination 'platform=iOS Simulator,name=iPhone XS Max' \
 testLanguage fr \
 test
```

The exported screenshots directory structure should be:

``` 
<export-folder>/testScreenshots/"iPhone 8 Plus"/[screenshots].png
<export-folder>/testScreenshots/"iPhone XS Max"/[screenshots].png
etc.
```

What do you think?

**Comment by abotkin-cpi** [comment]
@ntsh Agreed, this would be a good enhancement.  We’re also pulling screenshots and attachments from all the test failures as well which I know some folks would prefer not to get.

**Comment by abotkin-cpi** [comment]
@ntsh This should be resolved in xcparse 0.5.  Upgrade using `brew upgrade xcparse`.  We've added new options to the screenshots sub-command where you can specify if you want division by model, OS version, and/or test run configuration.

For your use case, you can use:

`xcparse screenshots --model Test.xcresult screenshotFolder`

The recommended use we suggest is this:

`xcparse screenshots --model --os --test-run Test.xcresult screenshotFolder`

This would also include division by OS version & test run configuration (using the name of your test plan's configuration's name)
